R/cranlogs.R

```diff
@@ -167,8 +167,9 @@ cran_top_downloads <- function(when = c("last-day", "last-week",
   when <- match.arg(when)
   req <- GET(paste0(top_url, when, '/', count))
   stop_for_status(req)
-  r <- fromJSON(content(req, as = "text"), simplifyVector = FALSE)
-
+  r <- fromJSON(content(req, as = "text", encoding = "UTF-8"), 
+    simplifyVector = FALSE)
```

Comment:
Thanks. In general you cannot know that the encoding is UTF-8, it is up to the server, and this is a bug in the server that sends the response. In this case we know that it is UTF-8, though, and also, it is just ASCII, so it does not matter much.

Reply:
I think I only saw the warning recently. Maybe changes upstream (e.g., 'httr').